### DISCUSSION TITLE
Autolaunch browser.sh at start?

### DISCUSSION THREAD

**Discussion by henroFall** [debloper/piosk#11] [Q&A]
Hey, I went in circles a bit trying to figure out why my Pi was booting to the desktop. I thought it was because I installed through SSH, but I don't see anywhere that you are setting up to call that at boot?

Am I missing something?

I'm installing onto a Pi 3, and flashed my SD card using the official Raspberry Pi flashing util, and selected the 32 bit build of Bookworm.

Thanks!

**Comment by debloper** [comment]
First of all, thanks for giving it a try, and taking your time to open up the discussion (and sorry for the late response - I had no idea anyone other than me have started using it).

Raspberry Pi OS by default autologins into desktop. That's not unusual behavior, and I'm not doing anything special. I'd guess RPi OS was designed to be newbie friendly (especially for people that are new to Linux). So, auto login to desktop is just a way to eliminate a layer of entry barrier.

This seems like a common question, and there are articles to explain how to disable it... like [this one](https://pimylifeup.com/raspberry-pi-os-disable-auto-login/). But in this usecase, you don't wanna do that.

I'm keeping the discussion open and pinning it, as I'd imagine many other may have questions along this line. Please mark it as answer, if it clears up your doubt. Thanks again for using PiOSK!
